apps/io-wallet-user-func/src/infra/pid-issuer/client.ts

```diff
@@ -51,6 +56,11 @@ export class PidIssuerClient
           method: "POST",
           signal: AbortSignal.timeout(this.#requestTimeout),
           ...this.#init,
+          headers: {
+            ...this.#init.headers,
+            Authorization: `Bearer ${accessToken}`,
+            "X-Client-Cert": btoa(this.#clientCertificate),
```

Comment:
add a comment about what this function does. What do you think about having the base64 certificate already in the environment variable?